scrapers/covid-19/govScrapers/getCanada.js

```diff
@@ -20,10 +21,18 @@ const canadaData = async () => {
 	try {
 		const canadaRes = (await axios.get('https://health-infobase.canada.ca/src/data/covidLive/covid19.csv')).data;
 		const parsedCanadaData = await csvUtils.parseCsvData(canadaRes);
-		return filterByDate(parsedCanadaData).map(province => ({
+		return parsedCanadaData.map(province => ({
 			updated: Date.now(),
 			province: province.prname === 'Canada' ? 'Total' : province.prname,
+			date: province.date,
+			todayCases: parseInt(province.numtoday) || 0,
+			todayTests: parseInt(province.numtestedtoday) || 0,
+			todayRecovered: parseInt(province.numrecoveredtoday) || 0,
+			todayDeaths: parseInt(province.numdeathstoday) || 0,
 			cases: parseInt(province.numconf) || 0,
+			active: parseInt(province.numactive) || 0,
+			tests: parseInt(province.numtested) || 0,
+			recovered: parseInt(province.numrecover) || 0,
```

Comment:
instead of 0s we should put `nulls` or nothing, otherwise we report incorrect data

Reply:
yeah true, I just went with what was already done by collin, but good idea